<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: white
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="7">
    <widget class="QToolButton" name="stop_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/stop.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QToolButton" name="next_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/next.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QToolButton" name="play_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/play.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="10">
    <widget class="QSlider" name="timeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QLabel" name="time">
     <property name="text">
      <string>0:0/0:0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="del_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/minus.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="add_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/plus.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QToolButton" name="pause_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/pause.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="11">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="previous_but">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/previous.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="13">
    <widget class="QLabel" name="currentTrack">
     <property name="text">
      <string>Current file</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QToolButton" name="sound">
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-image: url(:/buttons/sound.png);
background-position: bottom central
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="11">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
